src/sys/unix/tcp/listener.rs

```diff
@@ -13,6 +13,12 @@ pub struct TcpListener {
 }
 
 impl TcpListener {
+    pub(crate) fn from_std(inner: net::TcpListener) -> Self {
```

Comment:
You can just use `pub`, the type isn't exported from the crate.

Reply:
Same for the other types.

Reply:
I believe they using pub for pub(crate) is falling out of favor.  There is a lint (enabled in tokio) that warns on this.

Reply:
I'm fine with changing to `pub(crate)` to, as long as its consistent.

Reply:
The lint is `unreachable_pub`. I'm fine either leaving this here and fixing everything else in a follow up PR or switch to just `pub` in the PR here and then add the lint & fix.

Reply:
I'll remove from these fns as part of this PR. An addition of the lint and subsequent cleanup can be done in a separate PR!